models/comments.js

```diff
@@ -21,3 +21,25 @@ exports.selectCommentsByID = (article_id) => {
       return array.rows;
     });
 };
+
+exports.insertComments = (article_id, username, body) => {
+  const insertUser = db.query(
+    `INSERT INTO users (username, name) 
```

Comment:
Same as before, we only want to post a comment, not create a new user. I do like what you've done though!